tests/beaker_tests/accessvlan/accessvlan_provider_defaults.rb

```diff
@@ -45,118 +45,117 @@
+      UtilityLib.search_pattern_in_output(stdout, [%r{interface Ethernet1/4}],
+                                          false, self, logger)
     end
 
     logger.info("Setup switch for provider test :: #{result}")
   end
 
   # @step [Step] Requests manifest from the master server to the agent.
-  step "TestStep :: Get resource present manifest from master" do 
+  step 'TestStep :: Get resource present manifest from master' do
     # Expected exit_code is 0 since this is a bash shell cmd.
-    on(master, AccessVlanLib.create_accessvlan_manifest_present())
+    on(master, AccessVlanLib.create_accessvlan_manifest_present)
 
     # Expected exit_code is 2 since this is a puppet agent cmd with change.
     cmd_str = UtilityLib.get_namespace_cmd(agent, UtilityLib::PUPPET_BINPATH +
-      "agent -t", options)
-    on(agent, cmd_str, {:acceptable_exit_codes => [2]}) 
+      'agent -t', options)
+    on(agent, cmd_str, acceptable_exit_codes: [2])
 
     logger.info("Get resource present manifest from master :: #{result}")
   end
 
   # @step [Step] Checks cisco_interface resource on agent using resource cmd.
-  step "TestStep :: Check cisco_interface resource presence on agent" do 
+  step 'TestStep :: Check cisco_interface resource presence on agent' do
     # Expected exit_code is 0 since this is a puppet resource cmd.
     # Flag is set to false to check for presence of RegExp pattern in stdout.
     cmd_str = UtilityLib.get_namespace_cmd(agent, UtilityLib::PUPPET_BINPATH +
       "resource cisco_interface 'ethernet1/4'", options)
     on(agent, cmd_str) do
       UtilityLib.search_pattern_in_output(stdout,
-        {'ensure'                      => 'present',
-        'access_vlan'                  => '128',
-        'ipv4_proxy_arp'               => 'false',
-        'ipv4_redirects'               => 'true',
-        'negotiate_auto'               => 'true',
-        'shutdown'                     => 'false',
-        'switchport_autostate_exclude' => 'false',
-        'switchport_mode'              => 'access',
-        'switchport_vtp'               => 'false'},
-        false, self, logger)
+                                          { 'ensure'                       => 'present',
+                                            'access_vlan'                  => '128',
+                                            'ipv4_proxy_arp'               => 'false',
+                                            'ipv4_redirects'               => 'true',
+                                            'negotiate_auto'               => 'true',
+                                            'shutdown'                     => 'false',
+                                            'switchport_autostate_exclude' => 'false',
+                                            'switchport_mode'              => 'access',
+                                            'switchport_vtp'               => 'false' },
+                                          false, self, logger)
```

Comment:
I think I commented on this on the earlier diffs - it would probably be more readable to instead do:

```ruby
       UtilityLib.search_pattern_in_output(
         stdout,
         {
           'ensure'                      => 'present',
           'access_vlan'                  => '128',
           'ipv4_proxy_arp'               => 'false',
           'ipv4_redirects'               => 'true',
           'negotiate_auto'               => 'true',
           'shutdown'                     => 'false',
           'switchport_autostate_exclude' => 'false',
           'switchport_mode'              => 'access',
           'switchport_vtp'               => 'false',
         }, false, self, logger)
```

Similar comment throughout - however I know we're planning to refactor a lot of this code in the future so I don't think this is necessarily something that needs to be fixed right now.